### PR TITLE
feat(web)!: provide File instance

### LINKED ISSUE #25: feat: file should be an instance of Blob
On the web, the picker returns an object and optionally base64 string if `readData` is enabled. 

I could convert base64 to Blob but ideally it should just return a Blob like the native file picker which is easier to work with without any conversions required.

Thanks

### PR DESCRIPTION
**Pull request by reslear**
closes #25 

@robingenz is docs auto-generating?

### CODE REVIEW

File: src/definitions.ts [hunk 1]
```diff
@@ -30,10 +30,10 @@ export interface PickFilesOptions {
 }
 
 export interface PickFilesResult {
-  files: File[];
+  files: FileModel[];
```
Comment (robingenz) [inline src/definitions.ts:33]:
```suggestion
  files: FileResult[];
```

File: src/definitions.ts [hunk 2]
```diff
@@ -58,4 +58,13 @@ export interface File {
    * The size of the file in bytes.
    */
   size: number;
+
+  /**
+   * The File instance
```
Comment (robingenz) [inline src/definitions.ts:63]:
```suggestion
   * The File instance.
```

**Review by robingenz — CHANGES_REQUESTED** [review]
Thank you! 2 minor changes requested.

**Comment by robingenz** [comment]
> closes #25
> 
> @robingenz is docs auto-generating?

No, please run `npm run docgen` and `npm run fmt`.

**Comment by reslear** [comment]
@robingenz so, we change interface name, do we need changes for backward compatibility or use new version as major?

**Comment by robingenz** [comment]
> @robingenz so, we change interface name, do we need changes for backward compatibility or use new version as major?

Next version will be `0.4.0` which means that the version contains breaking changes, as we are still in major version zero (0.y.z).

**Comment by robingenz** [comment]
Hi @reslear,
I've reconsidered and decided that I don't want to release the next breaking changes until the update to Capacitor 4.
This plugin has received a lot of breaking changes and I would like to reduce that a bit.
For this reason I have created a new PR without breaking changes (see #37).
You can give it a try if you want:
```
npm i @robingenz/capacitor-file-picker@0.3.0-dev.d8fa252.1657736483
```

However, thank you for this PR! I hope that is fine for you.

**Pull request closed without merging**